app/templates/provider/moderation.hbs

```diff
@@ -0,0 +1,2 @@
+{{moderation-base provider-title='engrXiv' provider-id='engrxiv' unread-count=model.pendingPreprints.meta.pagination.total active='Moderation'}}
```

Comment:
You shouldn't need to add this components to these pages/routes. If you place them in `provider.hbs` with an `{{outlet}}` ember will render all subpages/subroutes in between them.